client/ios/Podfile

```diff
@@ -40,6 +40,9 @@ target 'bikelockerSpring2019' do
 
     pod 'rn-fetch-blob', :path => '../node_modules/rn-fetch-blob'
 
+
+    pod 'react-native-splash-screen', :path => '../node_modules/react-native-splash-screen'
+
```

Comment:
did you install splash screen package using pods?

Reply:
No I didn't